### ISSUE TITLE
Crashed when I was taking out 4K storage drive, from the disk drive

### ISSUE THREAD

**Issue by soerenkk** [raoulvdberge/refinedstorage#1292]
#### Issue description:
Crashed when I was taking out 4K storage drive, from the disk drive

#### What happens:
Crashed when I was taking out 4K storage drive, from the disk drive

#### What you expected to happen:
To pull out the 4K storage drive (to insert it into the disk manipulator)

#### Steps to reproduce:
Hard to say. All I did was taking it out, used normal mouse clicks, no shift clicking or other stuff.

#### Version (make sure you are on the latest version before reporting):

- Minecraft: 1.10.2
- Forge: 12.18.3.2254
- Refined Storage: 1.2.25

I'm using Reborn Storage (1.0.3.30) as well, if that can help you.

#### Side node:
I've noticed twice now, than when I enter the game, for the Refined Storage to know the recipes in the Reborn Storage multiblock thing, I have to replace a cable leading to the Reborn Storage multiblock thing.

Also, after reenter the map, I've lost the 4K storage drive with all of its content :'(

Beside the 4K storage drive there was a 64K fluid drive, other than that, no other drives.

Items not in drives are stored in Storage Drawers, using a External Storage on a Drawer Controller.

#### Does this issue occur on a server? [yes/no]
No

#### If a (crash)log is relevant for this issue, link it here:

[https://gist.github.com/soerenkk/c7099c4cbde92840f74e41c1ceb0b924]

**Comment by soerenkk** [comment]
By saying I've lost the 4K storage drive, I mean that the drive seems to be deleted. It wasn't thrown, like if you, craft in your inventory and exit the inventory while the items are in the grid. The drive is not in my inventory, or in the Refined Storage system either. It is as far as I can tell completely gone, like it has been deleted.

Additional, as far as I know, the game crashed when I've tried to put it in my inventory, it could had been the moment I've pulled it out, but I think I've mouse clicked it out and then "juggled" with it, when I've decided where to put it in my inventory.
